spec/routing/id_verification_routing_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+
+describe 'Identity Verification Routes', type: :routing do
+  context 'when FeatureManagement.enable_identity_verification? is false' do
+    before do
+      allow(Figaro.env).to receive(:enable_identity_verification).and_return('false')
+      Rails.application.reload_routes!
+    end
+
+    after(:all) do
+      Rails.application.reload_routes!
+    end
+
+    it 'does not route to any GET /verify/* endpoint' do
+      expect(get: '/verify').
```

Comment:
would it be more streamlined to put all of these verify routes into an array that we iterate over for each test csae (`page_not_found` and `be_routable`) ?